### PR TITLE
Rewrite error messages

### PR DESCRIPTION
**Pull request by sr5434**
I had to reopen #58664

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @sr5434 Second file _`tensorflow/python/keras/layers/advanced_activations.py`_ changes in this PR belongs to keras, hence please create PR in [kears](https://github.com/keras-team/keras) repo instead.  Thank you!

**Comment by sr5434** [comment]
> Hi @sr5434 Second file _`tensorflow/python/keras/layers/advanced_activations.py`_ changes in this PR belongs to keras, hence please create PR in [kears](https://github.com/keras-team/keras) repo instead. Thank you!

Hi @gbaned. The edit in _`tensorflow/python/keras/layers/advanced_activations.py`_ was from an old PR. That is why I removed the change in commit [d817115](https://github.com/tensorflow/tensorflow/pull/58665/commits/d8171151a12a291b959df152ba46a57aaf1a75ee). Thanks.

**Comment by sr5434** [comment]
@gbaned is everything alright?

File: tensorflow/python/keras/layers/advanced_activations.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ class LeakyReLU(Layer):
   >>> output = layer([-3.0, -1.0, 0.0, 2.0])
   >>> list(output.numpy())
   [-0.3, -0.1, 0.0, 2.0]
-
+
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/layers/advanced_activations.py:55]: Please revert changes in this file.

**Comment by mihaimaruseac** [comment]
Can you make one single commit with the change in the first file? 7 commits for just one single file change are a little bit too much.

You can squash all commits to a single one too if you don't want to open a separate PR.

**Comment by sr5434** [comment]
@mihaimaruseac thanks for the advice. Does everything look good?

**Review by mihaimaruseac — APPROVED** [review]

**Comment by sr5434** [comment]
When will this pull request get pulled?

**Comment by mihaimaruseac** [comment]
It got pulled in the Google system (see the 2 copybara checks). However, it seems internal build fails, probably some system is looking for the same error message (Hyrum's Law).

Someone will need to look at it internally and try to fix.

**Comment by sr5434** [comment]
@mihaimaruseac do you know when the internal tests will be fixed?

**Comment by mihaimaruseac** [comment]
It needs to be by someone on the team, owning the code.

**Comment by sr5434** [comment]
Thanks for the clarification. Do you know who owns this code?

**Comment by mihaimaruseac** [comment]
TFLite team. @gbaned please assign to someone from TFLite team (this and the other stale TFLite PRs, they cannot go in without manual shepherding)

**Review by alankelly — COMMENTED** [review]
These errors may be triggered my multiple issues. AllocateTensors can fail for many reasons, not just the reason in the error message.  With debug information enabled, clearer error messages are printed at the source of the error. These messages may end up being unhelpful and sending people in the wrong direction

Delegates application is really complicated and can fail for many reasons , if XNNPACK were to fail, sending the user to a gpu issue will just confuse them. I don't think this PR is suitable for merging.

**Comment by sr5434** [comment]
So there is no possibility of this PR getting merged?

**Comment by alankelly** [comment]
Sorry but as I said, the error messages in most cases but this specific case would serve to only confuse users.

**Comment by sr5434** [comment]
Ok.

**Pull request closed without merging**